charts/datadog/values.yaml

```diff
@@ -364,6 +364,28 @@ datadog:
     ## ref: https://docs.datadoghq.com/agent/logs/advanced_log_collection/?tab=configurationfile#automatic-multi-line-aggregation
     autoMultiLineDetection: false
 
+    ## Enable logs configurations related to multi-line defaults
+    autoMultiLineDefault:
```

Comment:
could you rename this field into `overrideAutoMultiLineDefault`
And add a "warning" in the description that said "these are advance settings that should not be used in most of the case"

Reply:
Is it possible to omit the advanced/undocumented settings from the default `values.yaml`? But leave them in the template that generates the `datadog-agent.yaml`? This would be more in-line with how the agent currently leaves these flags undocumented.

Reply:
Hi @gh123man 

agree, if we don't document it in the agent configuration file, we should not expose it as option in the helm chart. 

@gh123man could you list the options  that can be keep in this pull-request?

Reply:
@gh123man These values are templated to environment variables for the agent, not the `datadog-agent.yaml`.

@clamoriniere Is the warning necessary? I noticed the values.yaml doesn't contain any such warning on advanced settings.